recipes/biogeme-optimization/meta.yaml

```diff
@@ -36,7 +36,6 @@ about:
   home: http://biogeme.epfl.ch
   summary: Various optimization algorithms for teaching and research
   license: Custom
-  license_file: LICENSE
```

Comment:
If the license is not in the tarball, then you have to add it to this PR as an extra file in the recipe.